src/controllers/mainController.ts

```diff
@@ -160,14 +160,11 @@ export default class MainController implements vscode.Disposable {
                     vscode.commands.registerCommand(
                         Constants.cmdObjectExplorerNewQuery, async (treeNodeInfo: TreeNodeInfo) => {
                     const connectionCredentials = treeNodeInfo.connectionCredentials;
-                    if (connectionCredentials) {
-                        const databaseName = `${escapeCharacters(self.getDatabaseName(treeNodeInfo))}`;
-                        if (databaseName !== connectionCredentials.database) {
-                            connectionCredentials.database = databaseName;
-                        }
+                    const databaseName = `${escapeCharacters(self.getDatabaseName(treeNodeInfo))}`;
+                    if (databaseName !== connectionCredentials.database) {
```

Comment:
So this is actually a bug here - we shouldn't be escaping the characters (I'll file an issue separately)